webmails/roundcube/config.inc.php

```diff
@@ -13,6 +13,7 @@
 $config['zipdownload_selection'] = true;
 $config['enable_spellcheck'] = true;
 $config['spellcheck_engine'] = 'pspell';
+$config['session_lifetime'] = {{ SESSION_TIMEOUT // 60 }};
```

Comment:
This setting is missing in the roundcube deployed image. Perhaps the '// 60'  inflicts with the jinja scripting?
I will try removing it and  check if the config file is properly created.